src/OpenIddict.Client.WebIntegration/OpenIddictClientWebIntegrationProviders.xml

```diff
@@ -633,6 +633,24 @@
     </Environment>
   </Provider>
 
+  <Provider Name="MusicBrainz" Id="4aff779b-4d13-4e24-b5cb-5d798886ee40"
+            Documentation="https://musicbrainz.org/doc/Development/OAuth2">
+    <Environment Issuer="https://musicbrainz.org/">
+      <Configuration AuthorizationEndpoint="https://musicbrainz.org/oauth2/authorize"
+                     TokenEndpoint="https://musicbrainz.org/oauth2/token"
+                     UserinfoEndpoint="https://musicbrainz.org/oauth2/userinfo">
+        <GrantType Value="authorization_code" />
+        <GrantType Value="refresh_token" />
+      </Configuration>
+
+      <!--
+        Note: MusicBrainz requires specifying a scope on authorization, with "profile" being the most generic scope and thus the default
+      -->
+
+      <Scope Name="profile" Default="true" Required="true" />
```

Comment:
`Required="true"` is typically used when the userinfo endpoint cannot be accessed without the scope: do you confirm MusicBrainz' userinfo API returns an error if you don't request `profile`? (e.g `submit_isrc`)